.github/services/hdfs/hdfs_default_s3/action.yml

```diff
@@ -0,0 +1,71 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: hdfs_default_s3
+description: 'Behavior test for hdfs default over aws s3'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup java env
+      uses: actions/setup-java@v4
+      with:
+        distribution: temurin
+        java-version: "11"
+    - name: Load secrets
+      uses: 1password/load-secrets-action@v1
+      with:
+        export-env: true
+      env:
+        # Need a secret with this name to securely encrypt them - otherwise there will be warning
+        # See https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/CredentialProviderAPI.html#Keystore_Passwords
+        HADOOP_CREDSTORE_PASSWORD: op://services/hdfs/credstore_password
+        # TODO: Does this start with / and ends with /
+        OPENDAL_S3_ROOT: op://services/s3/root
+        OPENDAL_S3_BUCKET: op://services/s3/bucket
```

Comment:
Double checking:
- Does `op://services/s3/root` starts and ends with `/`?
- Does `op://services/s3/bucket ` starts and ends with `/`?

Reply:
> * Does `op://services/s3/root` starts and ends with `/`?

No sure. opendal hdfs service should handle this correctly.

> * Does `op://services/s3/bucket ` starts and ends with `/`?

No.